### PR TITLE
Fix bor docker and add CI

### PR DESCRIPTION
**Pull request by jdkanani**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,19 +16,38 @@ npm install
 ### Compile all contracts
 
 ```bash
+npm run template:process
```
Comment (arthcp) [inline README.md:19]: If this is generating bor genesis then it should not be should under **_Compile all contracts_**.
Should we move it under **_Start Matic child chain_**  or maybe an independent section altogether?

File: scripts/process-templates.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+
+const program = require('commander')
+const nunjucks = require('nunjucks')
+const glob = require('glob')
+const fs = require('fs')
+const path = require('path')
+
+program.version('0.0.1')
+program.option('-c, --child-chain-id <child-chain-id>', 'Child chain id', '15001')
```
Comment (arthcp) [inline scripts/process-templates.js:9]: lets keep default chain id as `80001` since that is mumbai chain id

Comment (jdkanani) [reply scripts/process-templates.js:9]: No. We should keep our local default.

File: README.md (hunk 1 above)
Comment (jdkanani) [reply README.md:19]: We need to process all templates (when the next PR will come). We will also process contracts for bor/main chain id.

**Pull request merged**